src/main/resources/db/migration/V1.6__Add_hidden_and_update_created_by.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE deck ADD hidden bit DEFAULT FALSE;
+
+UPDATE deck SET created_by = user_id WHERE deck_id > 4;
```

Comment:
user_id is an owner of the deck. But in fifth migration file someone added  created_by column to deck table and inserted value 1 by default to each deck. So, in "Cabiner" -> "Manage decks" we get a list of own deck by user_id, but we can`t edit the deck because permission checks by created_by value.

Reply:
Ok, thanks.